crates/core/src/model/scalar/date.rs

```diff
@@ -1,5 +1,6 @@
-use std::fmt;
-use std::ops;
+use std::{convert::TryInto, fmt, ops};
+
+type DateImpl = time::Date;
```

Comment:
nit: `DateImpl` was intentionally put below `Date` to make it so people would first see `pub` items

Reply:
Ahh ok, that makes sense, in my head I was putting it before the first usage in the pub struct.